### PR TITLE
License FAQ

### PR DESCRIPTION
**Pull request by dvaldivia**
Screenshot 2023-01-20 at 10 30 21 AM

Signed-off-by: Daniel Valdivia <18384552+dvaldivia@users.noreply.github.com>

### CODE REVIEW

File: portal-ui/src/screens/Console/License/FAQModal.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
+// This file is part of MinIO Console Server
+// Copyright (c) 2022 MinIO, Inc.
```
Comment (jinapurapu) [inline portal-ui/src/screens/Console/License/FAQModal.tsx:2]:
```suggestion
// Copyright (c) 2023 MinIO, Inc.
```

**Review by pjuarezd — DISMISSED** [review]

File: portal-ui/src/screens/Console/License/licenseSlice.ts [hunk 2]
```diff
@@ -0,0 +1,42 @@
+// This file is part of MinIO Console Server
+// Copyright (c) 2022 MinIO, Inc.
```
Comment (jinapurapu) [inline portal-ui/src/screens/Console/License/licenseSlice.ts:2]:
```suggestion
// Copyright (c) 2023 MinIO, Inc.
```

**Review by jinapurapu — CHANGES_REQUESTED** [review]

**Review by reivaj05 — DISMISSED** [review]

File: portal-ui/src/screens/Console/License/licenseSlice.ts (hunk 2 above)
Comment (allanrogerr) [reply portal-ui/src/screens/Console/License/licenseSlice.ts:2]: Great catch - can this be automated?

**Review by allanrogerr — APPROVED** [review]

File: portal-ui/src/screens/Console/License/licenseSlice.ts (hunk 2 above)
Comment (dvaldivia) [reply portal-ui/src/screens/Console/License/licenseSlice.ts:2]: We would need to write an action to check the year for new files.

Fixed @jinapurapu

**Review by reivaj05 — APPROVED** [review]

**Review by jinapurapu — APPROVED** [review]

File: portal-ui/src/screens/Console/License/licenseSlice.ts [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+export interface IAddPool {
+  faqModalOpen: boolean;
+}
+
+const initialState: IAddPool = {
+  faqModalOpen: false,
+};
+
+export const licenseSlice = createSlice({
+  name: "addPool",
```
Comment (kaankabalak) [inline portal-ui/src/screens/Console/License/licenseSlice.ts:28]: @dvaldivia Wouldn't we need to change the name of the slice here?

File: portal-ui/src/screens/Console/License/utils.tsx [hunk 4]
```diff
@@ -138,13 +141,25 @@ export const COMMUNITY_PLAN_FEATURES = [
             distribute the complete, corresponding source code of the combined
             work under the same <LicenseLink /> license. This requirement
             applies whether or not you modified MinIO.
+            <br />
+            <br />
+            <span
+              className="link-text"
+              onClick={() => {
+                console.log("ohla");
```
Comment (kaankabalak) [inline portal-ui/src/screens/Console/License/utils.tsx:149]: Could we also remove the `console.log` here?

File: portal-ui/src/screens/Console/License/LicenseFAQ.tsx [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+const LicenseFAQ = () => {
+  return (
+    <Fragment>
+      <h2>What is the GNU AGPL v3?</h2>
+      <p>
+        The GNU AGPL v3 is short for the "GNU Affero General Public License v3."
+        It is a common open source license certified by the Free Software
+        Foundation and the Open Source Initiative. You can get a copy of the GNU
+        AGPL v3 license with MinIO source code or at
+        https://www.gnu.org/licenses/agpl-3.0.en.html.
```
Comment (kaankabalak) [inline portal-ui/src/screens/Console/License/LicenseFAQ.tsx:28]: Would it make sense to turn this into a hyperlink as follows?

```diff
diff --git a/portal-ui/src/screens/Console/License/LicenseFAQ.tsx b/portal-ui/src/screens/Console/License/LicenseFAQ.tsx
index 3322d9f4..df9f9e65 100644
--- a/portal-ui/src/screens/Console/License/LicenseFAQ.tsx
+++ b/portal-ui/src/screens/Console/License/LicenseFAQ.tsx
@@ -24,8 +24,19 @@ const LicenseFAQ = () => {
         The GNU AGPL v3 is short for the "GNU Affero General Public License v3."
         It is a common open source license certified by the Free Software
         Foundation and the Open Source Initiative. You can get a copy of the GNU
-        AGPL v3 license with MinIO source code or at
-        https://www.gnu.org/licenses/agpl-3.0.en.html.
+        AGPL v3 license with MinIO source code or at{" "}
+        <a
+          href="https://www.gnu.org/licenses/agpl-3.0.en.html"
+          className={"link-text"}
+          target="_blank"
+          rel="noopener noreferrer"
+          style={{
+            color: "#2781B0",
+          }}
+        >
+          https://www.gnu.org/licenses/agpl-3.0.en.html
+        </a>
+        .
       </p>
       <h2>What does it mean for me to comply with the GNU AGPL v3?</h2>
       <p>

```

Comment (dvaldivia) [reply portal-ui/src/screens/Console/License/LicenseFAQ.tsx:28]: it does!

File: portal-ui/src/screens/Console/License/utils.tsx (hunk 4 above)
Comment (dvaldivia) [reply portal-ui/src/screens/Console/License/utils.tsx:149]: solved

File: portal-ui/src/screens/Console/License/licenseSlice.ts (hunk 3 above)
Comment (dvaldivia) [reply portal-ui/src/screens/Console/License/licenseSlice.ts:28]: solved

**Comment by kaankabalak** [comment]
Hi @dvaldivia, just a quick comment, seems like we need a space here between the word `at` and the link:

Screen Shot 2023-01-20 at 2 32 14 PM

**Review by kaankabalak — DISMISSED** [review]

File: portal-ui/src/screens/Console/License/LicenseFAQ.tsx [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+        to comply with the obligations specified in the AGPL v3. Otherwise, you
+        may risk infringing MinIO’s copyrights.
+      </p>
+      <h2>Making combined or derivative works of MinIO</h2>
+      <p>
+        Combining MinIO software as part of a larger software stack triggers
+        your GNU AGPL v3 obligations.
+      </p>
+      <p>
+        The method of combining does not matter. When MinIO is linked to a
```
Comment (allanrogerr) [inline portal-ui/src/screens/Console/License/LicenseFAQ.tsx:46]: The method of combining does not matter. When MinIO is linked to a
        larger software stack in any form, including statically, dynamically,
        pipes, containerized or invoked remotely, the AGPL v3 applies to
        your use. What triggers the AGPL v3 obligations is the exchanging of data
        between the larger stack and MinIO.

**Review by allanrogerr — COMMENTED** [review]
I would suggest these small grammar changes. @kaankabalak @dvaldivia

File: portal-ui/src/screens/Console/License/LicenseFAQ.tsx [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+        larger software stack in any form, including statically, dynamically,
+        pipes, or containerized and invoked remotely, the AGPL v3 applies to
+        your use. What triggers the AGPL v3 obligations is the exchanging data
```
Comment (allanrogerr) [inline portal-ui/src/screens/Console/License/LicenseFAQ.tsx:49]:
```suggestion
        pipes, containerized or invoked remotely, the AGPL v3 applies to
        your use. What triggers the AGPL v3 obligations is the exchanging of data
```

**Review by prakashsvmx — APPROVED** [review]
LGTM :+1: 
the website has been updated to redirect to GNU AGPL v3 as well.

**Review by bexsoft — CHANGES_REQUESTED** [review]

**Review by bexsoft — APPROVED** [review]

**Review by pjuarezd — APPROVED** [review]

**Pull request merged**